client/src/entrypoints/admin/modal-workflow.js

```diff
@@ -82,37 +82,37 @@ function ModalWorkflow(opts) {
     self.body = self.container.find('.modal-body');
   }
 
-  self.loadUrl = function (url, urlParams) {
+  self.loadUrl = function loadUrl(url, urlParams) {
     $.get(url, urlParams, self.loadResponseText, 'text').fail(errorCallback);
   };
 
-  self.postForm = function (url, formData) {
+  self.postForm = function postForm(url, formData) {
     $.post(url, formData, self.loadResponseText, 'text').fail(errorCallback);
   };
 
-  self.ajaxifyForm = function (formSelector) {
-    $(formSelector).each(function () {
+  self.ajaxifyForm = function ajaxifyForm(formSelector) {
+    $(formSelector).each(() => {
```

Comment:
Sorry to call you out but these changes broke the page chooser @lb-, I opened #12618 for more details. I think this PR is still useful in itself, but we just need to be more careful around changing `function` to arrow functions.

Reply:
https://github.com/wagtail/wagtail/pull/12619